requirements.txt

```
@@ -1,4 +1,5 @@
 Mako
+regex==2017.9.23
```

```suggestion
regex==2021.11.10
```
https://github.com/mrabarnett/mrab-regex 
Python 2 is no longer supported. The last release that supported Python 2 was 2021.11.10.